### DISCUSSION TITLE
The "Sponsor" button has different height with "Starred"

### DISCUSSION THREAD

**Discussion by ranistar** [community/community#52777] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

The "Sponsor" button has different height with "Starred":

![image](https://user-images.githubusercontent.com/14199700/231937402-c0504629-d924-4af0-8688-bef6be090b08.png)

and here:
![image](https://user-images.githubusercontent.com/14199700/231937617-53ff7562-748b-41f8-905e-53f516f88a0d.png)

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @ranistar ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
